hudi-flink/src/test/java/org/apache/hudi/table/HoodieDataSourceITCase.java

```diff
@@ -48,12 +50,13 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.TimeUnit;
 import java.util.stream.Collectors;
 
 import static org.apache.hudi.utils.TestData.assertRowsEquals;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
 
 /**
```

Comment:
We should never import `.*`. Please revert.

Reply:
My bad ,  forget to exec `mvn checkstyle`